external/farbtastic/README.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
```

Comment:
Shall we remove the readme.html and only keep the readme.md?

Reply:
Sure, will do.